config/config.yaml

```diff
@@ -132,3 +132,5 @@ preprocessorConfig:
   dataTransformer: ${PREPROCESSOR_TRANSFORMER:-json}
   rateLimiterMaxBurstSeconds: ${PREPROCESSOR_RATE_LIMITER_MAX_BURST_SECONDS:-1}
   kafkaPartitionStickyTimeoutMs: ${KAFKA_PARTITION_STICKY_TIMEOUT_MS:-0}
+  bootstrapServers: ${KAFKA_BOOTSTRAP_SERVERS:-localhost:9092}
+  useBulkApi: ${KALDB_PREPROCESSOR_USE_BULK_API:-false}
```

Comment:
Instead of a separate config here, can we just infer this based on the presence/absence of the stream configs / upstream topic?

Reply:
it just seems safer to keep an explicit key. We anyways want to move to this and eventually remove the pull based option.